{{cookiecutter.project_name}}/tests/test_token.py

```diff
@@ -172,24 +172,25 @@ def test_mint(token, owner, receiver):
     Create an approved amount of tokens.
     """
     totalSupply = token.totalSupply()
-    assert totalSupply == 1000
+    assert totalSupply == {{cookiecutter.premint_amount}}
 
     receiver_balance = token.balanceOf(receiver)
     assert receiver_balance == 0
 
-    tx = token.mint(receiver, 420, sender=owner)
+    TEST_AMOUNT = 420
+    tx = token.mint(receiver, TEST_AMOUNT, sender=owner)
 
     logs = list(tx.decode_logs(token.Transfer))
     assert len(logs) == 1
     assert logs[0].sender == ZERO_ADDRESS
     assert logs[0].receiver == receiver.address
-    assert logs[0].amount == 420
+    assert logs[0].amount == TEST_AMOUNT
 
     receiver_balance = token.balanceOf(receiver)
-    assert receiver_balance == 420
+    assert receiver_balance == TEST_AMOUNT
 
     totalSupply = token.totalSupply()
-    assert totalSupply == 1420
+    assert totalSupply == {{cookiecutter.premint_amount}} + TEST_AMOUNT
```

Comment:
because then down here you can do
```suggestion
    assert totalSupply == PREMINT_AMOUNT + TEST_AMOUNT
```